docs/src/getting_started/file_format.rst

```diff
@@ -329,6 +329,13 @@ left-multiplied onto the current one. The following choices are available:
 
       <matrix value="0 -0.53 0 -1.79 0.92 0 0 8.03 0 0 0.53 0 0 0 0 1"/>
 
+* Explicit 3x3 matrices in row-major order. Internally, this will be converted to a 4x4 matrix with the same last row and column as the identity matrix.
```

Comment:
Actually will the resulting `m[3][3] == 1.f`?

Reply:
Looking at the constructor used to cast a Matrix4f from a Matrix3f, yes